packages/calling/src/CallingClient/calling/callManager.ts

```diff
@@ -421,6 +429,21 @@ export class CallManager extends Eventing<CallEventTypes> implements ICallManage
   public getActiveCalls = (): Record<string, ICall> => {
```

Comment:
This also should be private

Reply:
In getConnectedCall and getActiveCalls in CallingClient we are calling this function